docs/src/image_loading.md

```diff
@@ -0,0 +1,143 @@
+* **Vendor Cfg Storage**: A volatile memory storage used to contain vendor specific configurations.
+* **SOC Images** SOC Image 1 is a firmware for Vendor CPU and loaded to Vendor RAM. SOC Image 2 is a configuration binary to be loaded to Vendor Cfg Storage.
+* **SOC Config** : A register accessible by the MCU ROM to select appropriate source (flash or BMC) for loading the SOC images.
+* **Caliptra 'Go' Wire** : A signal controlled by the Caliptra core routed to the reset line of the Vendor CPU.
+
+<p align="center">
+    <img src="images/image_loading_sample.svg" alt="flash_config" width="80%">
+</p>
+
+## Image Loading Steps
+
+The sequence diagram below shows the high level steps of loading MCU RT image and SOC images.
+
+* *Red Arrows indicates actions taken by Caliptra RT*
+* *Purple Arrows indicates actions taken by MCU ROM*
+* *Blue Arrows indicates actions taken by MCU RT*
+* *Black Arrows indicates actions taken by BMC FW*
+
+<p align="center">
+    <!--- https://www.plantuml.com/plantuml/uml/bLLlR_es47xlJt6_JffQgONrJpU95rDfrZ-LXfY0TPF6L9da0fQ2ZPnBM_hfvnmuYHlA-BsAxNly-EwvnyvDJj9IiSp4DrhW4Y5JymNDhLoBbGkeMAsa9ZWQ3oE8Z4xLlB2IbD578FFjIeYCPAPMP2K3_EGJsMCV-d7C23VUlHy6KcKpyrwY_3X43g8_EUY-hG2CH_s1ZxwKan0omjYyebt3K_UUONuXXDqE8ewwRnYupsI-u12Fmk0pwiHOfyvoPJHgObpiPq9eGmZbQPK-FHWLMYizhrPWD1o8BjHGLqUDSfCMe3gW1tVl61TbbzoYHN1VYpcIa0KPNIndQCMcTMV__60n-VVNEe-dfPmZZ4rXO_V136EQwcCkikirEX6xQZXbK0_w1IsCLHy8KYVmkv59ewW9tFzrOFqC5GA4QEGDEPcLMyY8Uz88N7JEE-V2R_U5-PJT0IYJDQJ6GkdbWTGgnPo4OpchKM7CBtkohGj3L2ivp7BSk29CpoKeUFaxwNGI270fMaNw-KPxrAHYIHZG7gRXrS4QiW4d6xTyqZ28jJJqrp647IZYLI3YuNnj5Q-5y3izatk9Ul3egC6_7sAFr7oxfGrLXgbcWjSSjcyS4-vX6IF1tTztKVTX2BUAMaY-zGy6tfH5o97Ah8FORqWO4oQm8G2Lxu3nEQoClzEWj4jD9yCq1PAp3CM-bsOYMeGZ9Alm5V-7jRPPm7_gLigEJVu0T1t1pJ-bsNNVv5bMOBFB3rKX0oIPI98GcSmTpWJSPY4oOrPiKoojnorIR0kVpBULbVEb14Bp0MIBbeVvY6wKpci3iq4QWH-X3jLlRmFmPJbDkBtiDBPKWGhAXrxyr0HnbKuHZwaV4dlL0UcVtsCk-KKb9tlyPwyxy8nMfUkDzPMkd3lLRueMWRyPCzKihMVVH_vpnvkxwKqFs97_BvtIM8NLa1TnZ7akjh7Gp3vMQx3pIXa-rQHGoJhLiR4MSpTBIdUJ-RgEQbDu0L4dKFfBtBXHiSp-0m00 -->
+    <img src="images/image_loading_sequence.svg" alt="flash_config" width="100%">
+</p>
+
+The following outlines the steps carried out by the MCU RT during the SOC boot process:
+
+1. MCU ROM reads a SOC Configuration register (implementation specific) to determine the source of the images to load (Flash/BMC).
+2. Caliptra RT authorizes and loads Caliptra RT (refer to [Caliptra Subsystem boot flow](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md#subsystem-boot-flow) for the detailed steps).
+3. Caliptra switches to Caliptra RT FW.
+4. Caliptra RT indicates to Recovery I/F that it is ready for the SOC manifest image (refer to [Caliptra Subsystem Recovery Sequence](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md#caliptra-subsystem-recovery-sequence) for the detailed steps).
+5. Retrieve SOC Manifest
+
+   1. If image is coming from BMC, BMC FW transfers manifest to Recovery I/F
```

Comment:
Minor:  It might be better to rephrase as "If image is coming from BMC, SoC manifest is retrieved from Recovery I/F",  since we don't need to define BMC FW behavior in our spec.

Reply:
I think you're right about not mentioning BMC FW. Although the for the Flash use case, the SoC manifest is also retrieved from Recovery I/F.

Should I just say BMC Recovery Agent and follow the wording in step 11 of https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md#subsystem-boot-flow ?

Reply:
yeah,  it's good to reword as `BMC's Recovery Agent`.